ionic2/app/services/api.service.ts

```diff
@@ -74,6 +74,16 @@ export class ApiService {
   }
 
   /**
+   * Get Pokemon by description.
+   * @param {string} description - Description of the Pokemon
+   * @returns {Observable<Pokemon[]>}
+   */
+  getPokemonByDescription(description: string): Observable<Pokemon[]> {
```

Comment:
@WoH I totally see your point in wating until it is merged :) I also dont see a use-case for searching for description, but we could aswell include it in the search bar. I think they just implemented it because ... who knows ... :)